src/pages/RegistrationPage.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+import { Div1, H1, P, Div2, Div3, Button, Button2 } from './Registration.styles';
+
+const RegistrationPage: FC = () => {
+  return (
+    <Div1>
+      <H1>Complete your registration</H1>
+      <P>Please select your role</P>
+      <Div2>
+        <P>I want to</P>
+        <Div3>
+          <Button>Work as a freelancer</Button>
```

Comment:
no, we connected i18n, in the layout, we should not use pure text only from constant